client/config/config.go

```diff
@@ -276,6 +276,16 @@ func (c *Config) FixupAndValidate() error {
 	} else {
 		return err
 	}
+
+	// XXX: Our SOCKS proxy client does not yet support UDP
```

Comment:
Can we add a link to a ticket to fix this issue?
// TODO(masala): Make SOCKS proxy client work with UDP. ticket number blah

Reply:
Tor doesn't support UDP transport either, so the fail-closed behavior is probably what we want.